src/actions/transactions.js

```diff
@@ -1,115 +1,51 @@
-
-  dispatch(loadingStarted(actionTypes.transactionsFilterSet));
-
-  return getTransactions({
-    networkConfig,
-    address,
-    limit,
-    filter,
-    customFilters,
-  }).then((response) => {
-    dispatch({
-      data: {
-        confirmed: response.data,
-        count: parseInt(response.meta.count, 10),
-        filter,
-        customFilters,
-      },
-      type: actionTypes.transactionsFiltered,
-    });
-    if (filter !== undefined) {
-      dispatch({
-        data: {
-          filterName: 'wallet',
-          value: filter,
-        },
-        type: actionTypes.addFilter,
-      });
-    }
-    dispatch(loadingFinished(actionTypes.transactionsFilterSet));
-  });
-};
-
-export const transactionsUpdateUnconfirmed = ({ address, pendingTransactions }) =>
-  (dispatch, getState) => {
-    const liskAPIClient = getState().peers.liskAPIClient;
-    return unconfirmedTransactions(liskAPIClient, address).then(response => dispatch({
-      data: {
-        failed: pendingTransactions.filter(tx =>
-          response.data.filter(unconfirmedTx => tx.id === unconfirmedTx.id).length === 0),
-      },
-      type: actionTypes.transactionsFailed,
-    }));
-  };
-
-export const loadTransactionsFinish = accountUpdated =>
-  (dispatch) => {
-    dispatch(loadingFinished(actionTypes.transactionsLoad));
-    dispatch({
-      data: accountUpdated,
-      type: actionTypes.transactionsLoadFinish,
-    });
-  };
-
-export const loadTransactions = ({ publicKey, address }) =>
-  (dispatch, getState) => {
-    const networkConfig = getState().network;
-    const lastActiveAddress = publicKey && extractAddress(publicKey);
-    const isSameAccount = lastActiveAddress === address;
-    dispatch(loadingStarted(actionTypes.transactionsLoad));
-    getTransactions({ networkConfig, address, limit: 25 })
-      .then((transactionsResponse) => {
-        dispatch(loadAccount({
-          address,
-          transactionsResponse,
-          isSameAccount,
-        }));
-        dispatch({
-          data: {
-            count: parseInt(transactionsResponse.meta.count, 10),
-            confirmed: transactionsResponse.data,
-          },
-          type: actionTypes.transactionsLoaded,
-        });
-      });
-  };
-
+/**
+ * This action is used to request transactions on dashboard and wallet page.
+ *
+ * @param {Object} params - all params
+ * @param {String} params.address - address of the account to fetch the transactions for
+ * @param {Number} params.limit - amount of transactions to fetch
+ * @param {Number} params.offset - index of the first transaction
+ * @param {Number} params.filter - one of values from src/constants/transactionFilters.js
+ * @param {Object} params.customFilters - object with filters for the filer dropdown
+ *   (e.g. minAmount, maxAmount, message, minDate, maxDate)
+ */
 export const transactionsRequested = ({
-  address, limit, offset, filter, customFilters = {},
+  address, limit, offset, filter, customFilters,
```

Comment:
we should have one object as filters instead of having filter and customFilters

Reply:
Done